discovery-kubernetes-api/src/main/scala/org/apache/pekko/discovery/kubernetes/Settings.scala

```diff
@@ -35,39 +36,61 @@ final class Settings(system: ExtendedActorSystem) extends Extension {
       if (hasDefined(key)) Some(config.getString(key)) else None
   }
 
-  private val kubernetesApi = system.settings.config.getConfig("pekko.discovery.kubernetes-api")
+  private val customSettings = configNameSpace.map(system.settings.config.getConfig)
```

Comment:
```suggestion
  private val namespace = configNameSpace.getOrElse("pekko.discovery.kubernetes-api")
  private val settings = system.settings.config.getConfig(namespace) 
```
What do you think? It forces the custom namespace to have complete settings, but has less code changes.

Reply:
I wanted to have access to the default configs as well, to have the ability to override certain properties and not have to give all of them, for example in my use case I am only gonna change the pod-namespace the rest are gonna be the same as the defaults.

Reply:
The com.typesafe:config library lets you fallback to other configs in another namespace - see the 'withFallback' method.

In HOCON, you can define configs that import config settings from another namespace.

Reply:
oh okay, let me update then

Reply:
done